src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -211,7 +211,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateSatelliteAssemblies EmbedResources="@(_SatelliteAssemblyResourceInputs)"
                                  AssemblyInfoFile="$(_AssemblyInfoFile)"
                                  OutputAssembly="$(_OutputAssembly)"
-                                 References="@(Reference)">
+                                 References="@(ReferencePath)"
```

Comment:
Yes, it should also depend on assembly info generation. Technically we build satellites late and get lucky but I'll be explicit